enterprise/migrations/0100_move_data_to_saved_for_later.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.13 on 2020-06-30 20:22
+
+from django.db import migrations
+# saved_for_later will replace the marked_done field
+def move_data_from_marked_done_to_saved_for_later(apps, schema_editor):
+    EnterpriseCourseEnrollment = apps.get_model('enterprise', 'EnterpriseCourseEnrollment')
+    for course_enrollment in EnterpriseCourseEnrollment.objects.all():
+        course_enrollment.saved_for_later = course_enrollment.marked_done
+
+
+def move_data_from_saved_for_later_to_marked_done(apps, schema_editor):
+    EnterpriseCourseEnrollment = apps.get_model('enterprise', 'EnterpriseCourseEnrollment')
+    for course_enrollment in EnterpriseCourseEnrollment.objects.all():
+        course_enrollment.marked_done = course_enrollment.saved_for_later
```

Comment:
I think you'll need to save the course_enrollment models after you edit them (for efficiency probably best to use https://docs.djangoproject.com/en/2.2/ref/models/querysets/#bulk-update)